contrib/google.golang.org/grpc/grpc.go

```diff
@@ -4,6 +4,7 @@
 package grpc // import "gopkg.in/DataDog/dd-trace-go.v1/contrib/google.golang.org/grpc"
 
 import (
+	"google.golang.org/grpc/codes"
```

Comment:
Whoops! This flew over here, can we move it back down?

Reply:
Fixed. Not sure how that happened… Thanks for noticing this!
It's a pity this project can't be used with `goimports -local …`. 😢